tests/micro/zephyr/test_zephyr.py

```diff
@@ -59,7 +66,7 @@ def _make_sess_from_op(model, zephyr_board, west_cmd, op_name, sched, arg_bufs):
 
 
 def _make_session(model, target, zephyr_board, west_cmd, mod):
-    test_name = f"{os.path.splitext(os.path.abspath(__file__))[0]}_{model}"
+    test_name = f"{os.path.splitext(os.path.abspath(__file__))[0]}_{zephyr_board}"
```

Comment:
why this change? seems like we should put both in the path, if anything

Reply:
@areusch  so we distinguish the test based on the zephyr_board. For ex. qemu_x86, qemu_riscv32 and qemu_riscv64 all use the same `model` which is `host`, but they have different zephyr_board. zephyr_board will always be unique.

Reply:
oh sorry, I was misreading. this looks good